deploy/assisted-service.yaml

```diff
@@ -104,6 +104,8 @@ spec:
               value: "info"
             - name: INSTALL_RH_CA
               value: "false"
+            - name: RH_QA_REG_CRED
```

Comment:
what is this?
From the name I don't understand the purpose

Reply:
Red Hat QA Registry Credentials.
Additional credentials needed in order to pull container images, that are not part of the regular pull secret.
In our case `registry.stage.redhat.io`

Reply:
Any suggestion for the name? 
What about REGISTRY_STAGE_CRED?

Reply:
Do we really need it as a secret? these are well-know internally creds for pulling images from registry.stage.redhat.io which is available only in VPN

Reply:
> Do we really need it as a secret?

Maybe not, I just make a habit of bringing it up when it looks like it might belong there.

As for the name, this doesn't necessarily need to be about stage or qa, it's just additional registry creds, so maybe just `REGISTRY_CREDS`?

Reply:
Update to `REGISTRY_CREDS`